xero_accounting.yaml

```diff
@@ -22752,6 +22745,54 @@ paths:
                           }
                         ]
                       }'
+  "/TaxRates/{TaxType}":
+    parameters:
+      - $ref: "#/components/parameters/requiredHeader"
+    get:
+      security:
+        - OAuth2: [accounting.settings, accounting.settings.read]
+      tags:
+        - Accounting
+      operationId: getTaxRates

```

Comment:
@jvcudis - can you update the operationId to "getTaxRateById"? This is how the other GETs by Id are labelled in the spec. Also, "getTaxRates" causes an error because it is already used as the operationId for GET /TaxRates

Reply:
@0GH4J - thanks for reviewing the PR 😄 
I updated the operationId value as requested.